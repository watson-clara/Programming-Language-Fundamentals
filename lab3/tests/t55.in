5
5
5
4
3
4
2
5
1
2
3
5
