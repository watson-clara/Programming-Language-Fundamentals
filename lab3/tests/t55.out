Welcome to Rock, Paper, Scissors, Lizard, Spock!

Please choose two players:
(1) StupidBot
(2) RandomBot
(3) IterativeBot
(4) LastPlayBot
(5) Human
Select player 1: Select player 2: Human vs. Human

Round 1:
(1) Rock
(2) Paper
(3) Scissors
(4) Lizard
(5) Spock
Enter your move: (1) Rock
(2) Paper
(3) Scissors
(4) Lizard
(5) Spock
Enter your move: Player 1 chose Spock
Player 2 chose Lizard
Lizard poisons Spock
Player 2 won the round

Round 2:
(1) Rock
(2) Paper
(3) Scissors
(4) Lizard
(5) Spock
Enter your move: (1) Rock
(2) Paper
(3) Scissors
(4) Lizard
(5) Spock
Enter your move: Player 1 chose Scissors
Player 2 chose Lizard
Scissors decapitate Lizard
Player 1 won the round

Round 3:
(1) Rock
(2) Paper
(3) Scissors
(4) Lizard
(5) Spock
Enter your move: (1) Rock
(2) Paper
(3) Scissors
(4) Lizard
(5) Spock
Enter your move: Player 1 chose Paper
Player 2 chose Spock
Paper disproves Spock
Player 1 won the round

Round 4:
(1) Rock
(2) Paper
(3) Scissors
(4) Lizard
(5) Spock
Enter your move: (1) Rock
(2) Paper
(3) Scissors
(4) Lizard
(5) Spock
Enter your move: Player 1 chose Rock
Player 2 chose Paper
Paper covers Rock
Player 2 won the round

Round 5:
(1) Rock
(2) Paper
(3) Scissors
(4) Lizard
(5) Spock
Enter your move: (1) Rock
(2) Paper
(3) Scissors
(4) Lizard
(5) Spock
Enter your move: Player 1 chose Scissors
Player 2 chose Spock
Spock smashes Scissors
Player 2 won the round

Final score is 2 to 3
Player 2 wins
